peril/rules/merge-on-green.ts

```diff
@@ -1,5 +1,5 @@
 import { danger, peril } from "danger"
-import * as octokit from "@octokit/rest"
+import octokit from "@octokit/rest"
```

Comment:
I'm not sure if changes to peril automation should be in this PR (not that we shouldn't do them at all, but it could be separate changeset).

Reply:
when i enabled moduleInterop this was required. These files wouldn't compile anymore. I just had a discussion with ward about moduleInterop and we came to a conclusion that it is worth having on.

The only changes in any of these peril files is dropping the `* as` from the imports.

Reply:
Hmmm, I'm not exactly sure how those things in Peril are run, but I don't think we control typescript config for it

I also don't have lot of context around `moduleInterop` setting - I just wonder if this change can actually break stuff depending on whatever ts config is used by Peril?

Reply:
To add to previous comment - I'm relatively sure it's not us who transpile this stuff, and actually .ts files get used and Peril handles those somehow (maybe using ts-node, maybe it transpiles them as part of its internal build setup when deploying updates)

Reply:
Seems like Peril transpiles on execution - it will run https://github.com/danger/danger-js/blob/dab0e63a97705617c0e0017d85a9ff7129cadd5f/source/runner/runners/utils/transpiler.ts#L142-L163

whenever it needs to execute commands - it also lazily grab code it needs to execute based on https://github.com/gatsbyjs/gatsby/blob/master/peril.settings.json - so it for sure grabs ts versions and then do it's own transpilation - I'm not sure if for us it uses `typescript` or babel with typescript preset/plugin, but I'm pretty sure we don't have any custom config for it.

Maybe it would be safer for now to disable whatever was failing for those files and not go down rabbit hole too much?

Reply:
It must use our tsconfig though, because when I made these changes, unit tests were failing and it was all the peril ones.

Reply:
Okay, so this is fixed now. I left the peril files unchanged. The solution was to make the peril folder have it's own package.json jest test with it's own jest configuration that set it's tsconfig file to have esModuleInterop: false. 

I dug into the options on the tools we used and wasn't finding any other options. The only other potential option on impl is to use jest projects, but that seemed like a larger refactor.